### PR TITLE
Pbi 26 logging

### PR DESCRIPTION
**Pull request by gurjaspalbedi**
1. Removed the large schema and replaced it with payload object. Before writing to project file payload is converted to individual properties (except the project property, it is not being logged because of huge size.)
2. Instead of whitelisting the events now we are blacklisting the events which should not be loggged.
3. New event for info icon
4. Logging for GraphiVariableType, collapsable table (data view) etc.

### CODE REVIEW

File: projects/dvl-fw/src/lib/plugins/activity-log/log-raw-data.ts [hunk 1]
```diff
@@ -57,23 +50,18 @@ export class ActivityLogRawData implements RawData {
...
-      fileName: get(msg, 'logData.data.payload.fileName'),
-      fileExtension: get(msg, 'logData.data.payload.fileExtension'),
-      createdUrl: get(msg, 'logData.data.payload.shareUrl'),
-      copiedUrl: get(msg, 'ng logData.data.payload.content'),
-      visualizationId: get(msg, 'logData.data.payload.id'),
-      graphicSymbolId: get(msg, 'logData.data.payload.symbol.recordStream.id'),
-      slot: get(msg, 'logData.data.slot'),
-      visualizationNumber: Number(get(msg, 'logData.data.payload')),
-      date : new Date().toLocaleString()
+      date : new Date().toLocaleString(),
```
Comment (bherr2) [inline projects/dvl-fw/src/lib/plugins/activity-log/log-raw-data.ts]: Please add time : new Date().getTime(). Epoch time is easier to analyze/post-process/sort.

Comment (gurjaspalbedi) [reply projects/dvl-fw/src/lib/plugins/activity-log/log-raw-data.ts]: @bherr2 Hi Bruce, I have made this change, are you reviewing the complete pull request? do I need another review from Gagan?

Comment (bherr2) [reply projects/dvl-fw/src/lib/plugins/activity-log/log-raw-data.ts]: Gagan should do the review and merge. I'm just lurking 😅

Comment (gurjaspalbedi) [reply projects/dvl-fw/src/lib/plugins/activity-log/log-raw-data.ts]: ok

**Pull request merged**